package.json

```diff
@@ -3,9 +3,10 @@
   "version": "1.0.0",
   "description": "Serverless API for shortening long links.",
   "scripts": {
-    "build": "sls package",
-    "deploy": "sls deploy",
-    "dev": "docker-compose up -d & sls offline start --reloadHandler",
+    "build": "tsc & sls package",
```

Comment:
I prefer not to extend the default script, because the name is lying, you name the script "build" but also run a Typescript compiler, and it can take a lot of time to just build packages.

Add one more script, with the name "build:lint": "tsc & sls package"